x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview.test.tsx

```diff
@@ -130,21 +127,49 @@ describe('EngineOverview', () => {
         expect(getTablePagination().pageIndex).toEqual(4);
       });
     });
+
+    describe('when on a platinum license', () => {
+      beforeAll(async () => {
+        mockApi.mockClear();
+        wrapper = await mountWithApiMock({
+          license: { type: 'platinum', isActive: true },
+          get: mockApi,
+        });
+      });
+
+      it('renders a 2nd meta engines table', () => {
+        expect(wrapper.find(EngineTable)).toHaveLength(2);
+      });
+
+      it('makes a 2nd call to the engines API with type meta', () => {
+        expect(mockApi).toHaveBeenNthCalledWith(2, '/api/app_search/engines', {
+          query: {
+            type: 'meta',
+            pageIndex: 1,
+          },
+        });
+      });
+    });
   });
 
   /**
    * Test helpers
    */
 
-  const mountWithApiMock = async ({ get }) => {
+  const mountWithApiMock = async ({ get, license }) => {
     let wrapper;
     const httpMock = { ...mockKibanaContext.http, get };
 
     // We get a lot of act() warning/errors in the terminal without this.
     // TBH, I don't fully understand why since Enzyme's mount is supposed to
     // have act() baked in - could be because of the wrapping context provider?
     await act(async () => {
-      wrapper = mountWithKibanaContext(<EngineOverview />, { http: httpMock });
+      wrapper = mountWithKibanaContext(
+        <LicenseContext.Provider value={{ license }}>
+          <EngineOverview />
+        </LicenseContext.Provider>,
```

Comment:
Depending on how often we use licensing context in the future, I might as a tech debt for some day just rename `mountWithKibanaContext` to `mountWithContext` and nest a `<LicenseContext.Provider>` in there by default

Reply:
That's a good point.